src/pretty_clif.rs

```diff
@@ -207,6 +208,10 @@ pub(crate) fn write_clif_file<'tcx>(
     mut clif_comments: &CommentWriter,
     value_ranges: Option<&cranelift_codegen::ValueLabelsRanges>,
 ) {
+    if !(cfg!(debug_assertions) || tcx.sess.opts.output_types.contains_key(&OutputType::LlvmAssembly)) {
```

Comment:
I have a slight preference for

```suggestion
    if !cfg!(debug_assertions) && !tcx.sess.opts.output_types.contains_key(&OutputType::LlvmAssembly) {
```